inbound.go

```diff
@@ -415,6 +420,13 @@ func (response *InboundCallResponse) doneSending() {
 	if span := response.span; span != nil {
 		if response.applicationError || response.systemError {
 			ext.Error.Set(span, true)
+			errorType := appErrorType
+			if response.systemError {
+				errorType = systemErrorType
+				// if the error is a system error, set the error code in span log
+				span.SetTag("rpc.tchannel.system_error_code", GetSystemErrorCode(response.err))
+			}
+			span.SetTag("rpc.tchannel.error_type", errorType)
```

Comment:
I feel something like this could be easier to read:
(Also - we can add the error message to the span log)
```
	if span := response.span; span != nil {
		if response.systemError {
			ext.Error.Set(span, true)
			span.SetTag("rpc.tchannel.error_type", "system")
			span.SetTag("rpc.tchannel.system.error_code", GetSystemErrorCode(response.err))
			span.LogKV("rpc.tchannel.system.error_message", GetSystemErrorMessage(response.err))
		} else if response.applicationError {
			ext.Error.Set(span, true)
			span.SetTag("rpc.tchannel.error_type", "application")
		}
```

Reply:
sorry for coming late to this. why weren't these changes tested?